main.css

```diff
@@ -477,6 +477,132 @@ body {
   border: 1px solid hsla(0, 100%, 50%, 0.48);
 }
 
+/* Modal design for modile */
+
+.modal {
+  width: 315px;
+  height: 815px;
+  padding: 20px 10px;
+  display: flex;
+  flex-direction: column;
+  border: 1px solid hsl(218, 25%, 35%);
+  border-radius: 8px;
+  position: fixed;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%) scale(0);
+  transition: 200ms ease-in-out;
+  background-color: hsl(0, 0%, 100%);
+  z-index: 9999;
+  overflow: auto;
+}
+
+.modal.activeModal {
+  transform: translate(-50%, -50%) scale(1);
+}
+
+.works .modal .closeBtnDiv {
+  position: absolute;
+  inset: 3% 0 0 85%;
+}
+
+.works .modal .closeBtnDiv .closeBtn {
+  width: 15px;
+  height: 15px;
+  background-image: url('images/x.png');
+  opacity: 0.8;
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-color: transparent;
+  border: none;
+  cursor: pointer;
+}
+
+.modal .imgDesk {
+  display: none;
+}
+
+.works .modal h2 {
+  font-style: normal;
+  font-weight: 600;
+  font-size: 32px;
+  line-height: 44px;
+  color: hsl(218, 54%, 20%);
+}
+
+.works .modal .tech {
+  list-style: none;
+  display: flex;
+  flex-wrap: wrap;
+  width: 100%;
+  padding: 20px 0;
+  gap: 10px;
+  align-items: flex-start;
+}
+
+.works .modal .tech .tech-item {
+  color: hsl(218, 25%, 35%);
+  font-style: normal;
+  font-weight: 600;
+  font-size: 12px;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+  padding: 8px 12px;
+  background-color: hsl(147, 11%, 83%);
+  border-radius: 4px;
+}
+
+.works .modal p {
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: hsl(218, 31%, 30%);
+}
+
+.works .modal .divBtn {
+  width: 100%;
+  padding-top: 15px;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.works .modal .divBtn .btnLive,
+.works .modal .divBtn .btnSource {
+  width: 141px;
+  padding: 8px 15px;
+  background-color: hsl(155, 54%, 46%);
+  border-radius: 4px;
+  color: hsl(0, 0%, 100%);
+  border: none;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  cursor: pointer;
+  text-align: center;
+  text-decoration: none;
+}
+
+#overlay {
+  position: fixed;
+  opacity: 0;
+  transition: 200ms ease-in-out;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  backdrop-filter: blur(10px);
+  pointer-events: none;
```

Comment:
- [ ] Good job adding the blur effect to the overlay, however as also requested by the previous reviewer [here](https://github.com/imranhossainemi/portfolio/pull/16/files#r1093294547), in order to stick to the Figma design you have respect the margin top and bottom of the popup, for that I can suggest  making the container or overlay scrollable and then adjust the popup inside to meet the the Figma standard :recycle:

Reply:
please https://github.com/imranhossainemi/portfolio/pull/16/files#r1094167116 check this

Reply:
@GedeonTS please provide me your test tool details

Reply:
Hi @mukhammadrizooff , I have even cloned your project and went to this branch and got almost the same result as the previous reviewer

Reply:
![Screenshot from 2023-02-02 12-45-04](https://user-images.githubusercontent.com/97834160/216310350-5da692b1-8ee9-4c0d-86de-0009379e4a72.png)

Reply:
@GedeonTS  please provide me your test encouragement details .

Reply:
> Hi @mukhammadrizooff , I have even cloned your project and went to this branch and got almost the same result as the previous reviewer

Why tag him

Reply:
> ![Screenshot from 2023-02-02 12-45-04](https://user-images.githubusercontent.com/97834160/216310350-5da692b1-8ee9-4c0d-86de-0009379e4a72.png)

@GedeonTS Please provide details about your test environment . Like test tool name , version , test machine os and version.